ember/app/components/plane-label-overlay.js

```diff
@@ -15,8 +15,10 @@ export default Ember.Component.extend({
 
     let badgeStyle = `display: inline-block; text-align: center; background: ${this.get('flight.color')}`;
 
+    let id = this.getWithDefault('flight.competition_id', '');
+    if (id == null) id = this.getWithDefault('flight.registration', '');
```

Comment:
you can simplify this to:

```js
let id = this.get('flight.competition_id') || this.get('flight.registration') || '';
```

the same should also work for the other change above.